src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/ThemeServiceTests.cs

```diff
@@ -1441,55 +1358,649 @@ await using (var context = InMemoryContentDbContext(contextId))
             };
             var publicationB = new Publication
             {
                 Title = "Publication B",
+                Slug = "publication-b",
+                SupersededBy = supersedingPublication,
             };
-            var publicationC = new Publication
+
+            var theme = new Theme
             {
-                Title = "Publication C"
+                Title = "Theme A",
+                Summary = "Theme A summary",
+                Topics = new List<Topic>
+                {
+                    new()
+                    {
+                        Title = "Topic A",
+                        // Publications are in random order
+                        // to check that ordering is done by title
+                        Publications = ListOf(publicationB, publicationA)
+                    },
+                },
+            };
+
+            var releaseFileA = new ReleaseFile
+            {
+                Release = new Release
+                {
+                    Publication = publicationA,
+                    ReleaseName = "2020",
+                    TimePeriodCoverage = TimeIdentifier.CalendarYear,
+                    Type = ReleaseType.OfficialStatistics,
+                    Published = new DateTime(2020, 1, 1),
+                },
+                File = new File { Type = FileType.Data },
+            };
+            var releaseFileB = new ReleaseFile
+            {
+                Release = new()
+                {
+                    Publication = publicationB,
+                    ReleaseName = "2020",
+                    TimePeriodCoverage = TimeIdentifier.CalendarYear,
+                    Type = ReleaseType.NationalStatistics,
+                    Published = new DateTime(2020, 2, 1),
+                },
+                File = new File { Type = FileType.Data },
+            };
+
+            var contextId = Guid.NewGuid().ToString();
+            await using (var context = InMemoryContentDbContext(contextId))
+            {
+                await context.AddRangeAsync(theme, releaseFileA, releaseFileB);
+                await context.SaveChangesAsync();
+            }
+
+            await using (var context = InMemoryContentDbContext(contextId))
+            {
+                var service = BuildThemeService(context);
+
+                var result = await service.GetPublicationTree(PublicationTreeFilter.DataTables);
+
+                Assert.Single(result);
+                Assert.Equal("Theme A", result[0].Title);
+
+                Assert.Single(result[0].Topics);
+                Assert.Equal("Topic A", result[0].Topics[0].Title);
+
+                var publications = result[0].Topics[0].Publications;
+                Assert.Single(publications);
+                Assert.Equal("Publication A", publications[0].Title);
+            }
+        }
+
+        [Fact]
+        public async Task GetPublicationTree_DataCatalogue_SingleThemeTopic()
```

Comment:
These test cases are a little inconsistent as we don't seem to have the `SingleThemeTopic` on some of the other filter types. In the majority of test cases, it seems like we assert to a single theme/topic so I'm not sure if this case adds an value as it's covered many times over.